docs/conf.py

```diff
@@ -47,8 +47,11 @@
 master_doc = "index"
 
 # General information about the project.
+import datetime
+
+year = datetime.datetime.now(tz=datetime.timezone.utc).date().year
 project = "Folium"
-copyright = "2013, Rob Story"
+copyright = f"{year}, Rob Story"
```

Comment:
A copyright statement with a single year indicates the year a document was first published. Better is "2013-{year}".

Reply:
Yep. Let's do it.